Tests/UnitTests/SubscriberAttributes/__Snapshots__/BackendSubscriberAttributesTests/iOS17-testPostReceiptWithAdServicesToken.1.json

```diff
@@ -15,9 +15,17 @@
       "fetch_token" : "YW4gYXdlc29tZSByZWNlaXB0",
       "initiation_source" : "restore",
       "is_restore" : false,
-      "observer_mode" : false
+      "observer_mode" : false,
```

Comment:
This is just an example of what it's going to look like. I have to actually write tests that include this.